src/Modules/ModParser.lua

```diff
@@ -5661,6 +5661,16 @@ local jewelOtherFuncs = {
 			out:NewMod("PassiveSkillEffect", "INC", 50, data.modSource)
 		end
 	end,
+	["75% increased Effect of non-Keystone Passive Skills in Radius"] = function(node, out, data)
```

Comment:
Use a regex instead of hardcoding percentages:

```lua
	["(%d+)%% increased Effect of non%-Keystone Passive Skills in Radius"] = function(num)
		return function(node, out, data)
			if node and node.type ~= "Keystone" and node.type ~= "ClassStart" then
				out:NewMod("PassiveSkillEffect", "INC", num, data.modSource)
			end
```